next-level/lib/checkers/go.py

```
@@ -0,0 +1,123 @@
+        result["comments_stripped"] = strip_result.get("stripped", 0)
+    except (ImportError, Exception):
+        result["comments_stripped"] = 0
+
+    # Find Go module root
+    module_root = _find_go_module_root(filepath)
+
+    # Lint with go vet
+    if module_root and shutil.which("go"):
+        try:
+            proc = subprocess.run(
+                ["go", "vet", "./..."],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)